dev-python/latexcodec/latexcodec-2.0.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Can we do EAPI=7 here? 

Unless there's a good reason (like eclasses), new packages and such should be the latest EAPI.

Reply:
Updated to EAPI=7.